keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -177,8 +171,6 @@ def __init__(
         lowercase: bool = True,
         strip_accents: bool = True,
```

Comment:
Not about the change, but I think we could standardize the spacing here?

Reply:
You mean the spacing around the `=`? I think that's actually the way `black` does it. If you have a type annotation, spaces around `=`, if not not, no space.